client/src/components/ModalVideo.js

```diff
@@ -1,33 +1,108 @@
-import React from "react";
+import React, { useState } from "react";
 import Button from "react-bootstrap/Button";
 import Modal from "react-bootstrap/Modal";
+import Nav from "react-bootstrap/Nav";
+import Tab from "react-bootstrap/Tab";
+import ReactPlayer from "react-player";
+
 import VideoCard from "./VideoCard";
 
-function ModalVideo({ title, url, location, show, handleClose }) {
+function ModalVideo({
+	title,
+	url,
+	audioUrl,
+	description,
+	show,
+	handleClose,
+	location,
+}) {
+	const [activeTab, setActiveTab] = useState("video");
+
+	const handleTabChange = (tab) => {
+		setActiveTab(tab);
+	};
+
+	let modalContent;
 
-  return (
-    <>
+	if (url && description) {
```

Comment:
Generally if statements this large are a red flag - this much logic should either be simplified or split into another component.

Reply:
Can you take a look at how to simplify this? It's currently 50 lines of logic and JSX.

Could you use more ternary statements like you have on lines 68-70 to make things optional?